src/secure-handlebars.js

```diff
@@ -33,18 +33,31 @@ function overrideHbsCreate() {
     var h = hbsCreate(),
         c = h.compile, 
         pc = h.precompile,
-        i, filterName;
+        i, filterName, parser;
+
+    // expose the Context Parser Handlebars
+    h.getCurrentContextParserHandlebars = function() {
+        return h.contextParserHandlebars;
+    };
```

Comment:
is this used anywhere?
ContextParserHandlebars is already exposed at https://github.com/yahoo/secure-handlebars/pull/120/files#diff-7744afc9654785f847f9d3e31548ad70R109